### DISCUSSION TITLE
Paid Windows version

### DISCUSSION THREAD

**Discussion by just-another-alien** [andrew-wilkes/logic-simulator-2#4] [General]
Hey uh, it is me again ( in case you did not notice me on your first logic simulator ),
I just saw that you made a new Version ( this one ) and noticed, that the v2.0.10 versions for mac and linux are free while the one for windows is not. ( not talking about the LogicSimX-turbo-v2.1.4-*.zip versions ). The thing is, that anyone who got the Godot Editor installed and can open this repository can just download the repository and make a build themself. ( I tested it and it works for me ).

BTW I love the design of your logic simulator and i am making one myself too but i was wondering if i could somehow help with yours?

Have a good day!
 - JustMindfulYT

**Comment by andrew-wilkes** [comment]
Hello, yes people can build it themselves but it is a hassle to do so and not made clear. I think that it's useful to have it open source so people can report bugs or suggest improvements. Thanks for the offer of help but I think that I am done adding features now. But I may add mods/tools.

Good luck with your one.

**Reply by just-another-alien** [reply]
Okay, good to know! Thank you, I'll need the luck :D

BTW I hope it is okay, that I made a build myself as my card is not working. I love the Simulation and I'll buy / donate when my card works again!

Have a good day!
JustMindfulYT - Damaged Engineer Interactive
